.vscode/launch.json

```diff
@@ -37,7 +37,17 @@
             "program": "${workspaceFolder}/agent/cmd/crane"
         },
     
-    
+        {
+            "name": "Launch cli",
```

Comment:
minor; but `CLI`

Reply:
Nice catch, CLI is an abbreviation. Fixed.